training-export-service/train_yolo.py

```
@@ -0,0 +1,124 @@
+    print('[train] training complete', flush=True)
+
+    # 5. Evaluate on the held-out test split (per-class mAP)
+    try:
+        metrics = model.val(
+            data=yaml_path, split='test', project='/runs',
+            name=f'{RUN_NAME}_test', device=device, exist_ok=True,
+        )
+        names = metrics.names if hasattr(metrics, 'names') else {}
+        per_class = {}
+        try:
+            for i, c in enumerate(metrics.box.ap_class_index):
```

Ruff (0.15.17)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)